app/navigation/Navigator.scala

```diff
@@ -275,6 +281,42 @@ class Navigator @Inject() () {
       .map(if (_) yesCall else noCall)
       .getOrElse(controllers.routes.JourneyRecoveryController.onPageLoad())
 
+  private def whichIdPage(ua: UserAnswers): Call =
+    ua.get(WhichIdentificationNumbersPage).fold(controllers.routes.JourneyRecoveryController.onPageLoad()) {
+      case set if set.contains(UTR) => routes.WhatIsUniqueTaxpayerReferenceController.onPageLoad(NormalMode)
```

Comment:
what if user selected UTR + CRN ?

Reply:
if selecting both that set will still contain UTR so we want the first page the user to go to be WhatIsUniqueTaxpayerReference